react/src/components/toggle/SprkToggle.js

```diff
@@ -9,10 +9,12 @@ import 'focus-visible';
 class SprkToggle extends Component {
   constructor(props) {
     super(props);
-    const { isDefaultOpen } = this.props;
+    // TODO: Remove isDefaultOpen in issue #1296
+    const { isDefaultOpen, isOpen = isDefaultOpen } = this.props;
+
     this.state = {
-      isOpen: isDefaultOpen || false,
-      height: isDefaultOpen ? 'auto' : 0,
+      isOpen: isOpen || false,
```

Comment:
Copying the prop into state in the constructor means any changes to the prop after init will be ignored.